src/components/I18nBanner.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import locale from 'browser-locale';
+
+import CaretDownSVG from 'svg/CaretDown';
+
+const languages = [
+  {
+    title: 'English',
+    abbr: 'en',
+  }, {
+    title: 'Español',
+    abbr: 'es',
+  }, {
+    title: 'Tiếng Việt',
+    abbr: 'vn',
+  }, {
+    title: '中文',
+    abbr: 'zh',
+  }, {
+    title: 'Chinese Traditional',
+    abbr: 'zh',
+  }, {
+    title: 'عربى',
+    abbr: 'ar',
+  }, {
+    title: '한국어',
+    abbr: 'ko',
+  }, {
+    title: 'اردو',
+    abbr: 'ur',
+  }, {
+    title: 'မြန်မာ',
+    abbr: 'my',
+  },
+]
+
+class I18nBanner extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeLanguage: locale().substring(0,2) || 'en', // we could
```

Comment:
could we?

Reply:
![image](https://user-images.githubusercontent.com/5697474/34243413-3de54636-e5e6-11e7-834d-0851baf2d211.png)